scripts/ci/index_ref_doc.py

```diff
@@ -44,7 +44,8 @@
 
     candidates_sorted = sorted(filtered_exts, key=lambda c: parse_version(c['metadata']['version']), reverse=True)
     chosen = candidates_sorted[0]
-    ALL_TESTS.append((extension_name, chosen['downloadUrl'], chosen['filename']))
+    if extension_name != 'partnercenter':
+        ALL_TESTS.append((extension_name, chosen['downloadUrl'], chosen['filename']))
```

Comment:
This gives better readability:

```suggestion
    if extension_name == 'partnercenter':
        continue

    ALL_TESTS.append((extension_name, chosen['downloadUrl'], chosen['filename']))
```